stream-chat-android-compose/api/stream-chat-android-compose.api

```diff
@@ -179,7 +179,7 @@ public final class io/getstream/chat/android/compose/state/messages/attachments/
 	public final fun copy (Landroidx/compose/ui/Modifier;Lio/getstream/chat/android/compose/state/messages/items/MessageItem;Lkotlin/jvm/functions/Function1;)Lio/getstream/chat/android/compose/state/messages/attachments/AttachmentState;
 	public static synthetic fun copy$default (Lio/getstream/chat/android/compose/state/messages/attachments/AttachmentState;Landroidx/compose/ui/Modifier;Lio/getstream/chat/android/compose/state/messages/items/MessageItem;Lkotlin/jvm/functions/Function1;ILjava/lang/Object;)Lio/getstream/chat/android/compose/state/messages/attachments/AttachmentState;
 	public fun equals (Ljava/lang/Object;)Z
-	public final fun getMessage ()Lio/getstream/chat/android/compose/state/messages/items/MessageItem;
+	public final fun getMessageItem ()Lio/getstream/chat/android/compose/state/messages/items/MessageItem;
```

Comment:
Might be worth adding an entry into the changelog

Reply:
Good point!